Sources/LambdaConverters.Wpf/SelectorErrorStrategy.cs

```
@@ -0,0 +1,21 @@
+﻿using System.Windows.Controls;
+using System.Windows.Data;
+
+namespace LambdaConverters
+{
+    /// <summary>
+    /// Defines the selector error strategy.
+    /// </summary>
+    public enum SelectorErrorStrategy
+    {
+        /// <summary>
+        /// Null is returned.
+        /// </summary>
+        ReturnDefaultValue,
```

_Default value_ is misleading as well. Default value of a reference type is always null. Please rename the enum field to `ReturnNull` (as you've already commented).